packages/app/src/app/pages/Sandbox/SearchDependencies/DependencyHit/elements.js

```diff
@@ -1,21 +1,35 @@
 import styled from 'styled-components';
 import Select from '@codesandbox/common/lib/components/Select';
 import { UserWithAvatar } from '@codesandbox/common/lib/components/UserWithAvatar';
+import Color from 'color';
+
+import GitHubLogo from 'app/components/GitHubLogo';
+
+export const GitHubLogoStyled = styled(GitHubLogo)`
+  color: ${props =>
+    !props.theme.light ? 'inherit' : 'rgba(255, 255, 255, 0.8)'};
+`;
 
 export const Container = styled.div`
   display: flex;
-  background: ${props =>
-    props.highlighted
-      ? props.theme.background2.darken(0.3)
-      : props.theme.background2};
-  color: ${props => props.theme.white};
+  color: ${props =>
```

Comment:
This one needs the `props.highlighted` to allow arrow key navigation. I'll put it back!